tools/permissions.py

```diff
@@ -0,0 +1,46 @@
+# This file is part of the EESSI build-and-deploy bot,
+# see https://github.com/EESSI/eessi-bot-software-layer
+#
+# The bot helps with requests to add software installations to the
+# EESSI software layer, see https://github.com/EESSI/software-layer
+#
+# author: Thomas Roeblitz (@trz42)
+#
+# license: GPLv2
+#
+import sys
+
+from pyghee.utils import log
+from tools import config
+
+BOT_CONTROL = "bot_control"
+COMMAND_PERMISSION = "command_permission"
+
+
+def check_command_permission(account):
```

Comment:
Checking of `build` and `deploy` permission should also come from here (eventually)?

Reply:
yes makes sense